linters/import/README.md

```diff
@@ -0,0 +1,53 @@
+# Import
+
+Import is a Golang linter that detects misordered imports given a certain pattern set.
+
+For example, let's say you have the following pattern set for your imports:
```

Comment:
nit: break the example into the args you'd give to the CLI tool, along with a sample import block that passes/fails